src/components/play.js

```diff
@@ -0,0 +1,78 @@
+const infoButton = document.querySelector('.info-button');
+const modeLists = document.querySelectorAll('.mode-list li');
+const sidebarToggleButton = document.querySelector('.mode-button');
+const modeSidebar = document.querySelector('.mode-sidebar');
+const homeButton = document.querySelector('.home-button');
+const finishButton = document.querySelector('.finish-button');
+const urlParams = new URLSearchParams(window.location.search);
+const guideMode = urlParams.get('guideMode');
+const soundButton = document.querySelector('.sound-button');
+const mainBgm = document.querySelector('.main-bgm');
```

Comment:
관련된 요소들끼리 묶어서 줄바꿈해주시면 가독성이 더 높아질 것 같네요.
```suggestion
const homeButton = document.querySelector('.home-button');
const sidebarToggleButton = document.querySelector('.mode-button');
const infoButton = document.querySelector('.info-button');
const soundButton = document.querySelector('.sound-button');
const finishButton = document.querySelector('.finish-button');

const modeSidebar = document.querySelector('.mode-sidebar');
const modeLists = document.querySelectorAll('.mode-list li');

const mainBgm = document.querySelector('.main-bgm');

const urlParams = new URLSearchParams(window.location.search);
const guideMode = urlParams.get('guideMode');
```
임의로 나눠봤습니다!